src/main/java/org/mastodon/views/trackscheme/LineageTreeLayoutImp.java

```diff
@@ -87,6 +90,11 @@ public class LineageTreeLayoutImp implements LineageTreeLayout
 
 	protected final SelectionModel< TrackSchemeVertex, TrackSchemeEdge > selection;
 
+	protected final GraphColorGenerator< TrackSchemeVertex, TrackSchemeEdge > colorGenerator;
+
+	@Nonnull
```

Comment:
Please remove the @Nonnulll annotation for the sake of consistency. All other fields are expected to be not null too.